stdlib/3/urllib/request.pyi

```diff
@@ -68,14 +68,15 @@ class Request:
     def header_items(self) -> List[Tuple[str, str]]: ...
 
 class OpenerDirector:
+    addheaders: List[Tuple[str, str]] = ...
```

Comment:
Strictly speaking this should not have `= ...` because it does not exist on the class, only on instances.

Reply:
Ah, I didn't realize that's how the distinction manifested in the stubs. In that case, it seems that a number of `Request`'s attributes should also not have `= ...` (all)?

https://github.com/python/cpython/blob/3.6/Lib/urllib/request.py#L324

Reply:
Yes, probably most (I didn't look through all of them).

I'm not sure type checkers currently implement this distinction, but they are supposed to. `x: int` in class scope means "instance variable only";  `x: int = ...` means "instance variable, with a default value on the class", and `x: ClassVar[int]` means "class variable only".